src/generators/docker-compose/dockercomposepeer.yaml.ts

```diff
@@ -78,7 +78,7 @@ ${this.options.org.peers
       - ${this.options.networkRootPath}/organizations/peerOrganizations/${this.options.org.fullName}/peers/${peer.name}.${this.options.org.fullName}/msp:/etc/hyperledger/fabric/msp
       - ${this.options.networkRootPath}/organizations/peerOrganizations/${this.options.org.fullName}/peers/${peer.name}.${this.options.org.fullName}/tls:/etc/hyperledger/fabric/tls
       - ${peer.name}.${this.options.org.fullName}:/var/hyperledger/production
-${this.options.ips.length > 0 ? `
+${this.options.ips ? `
```

Comment:
does it returns true if the array is empty ? 
ips: []

Reply:
if the ips section does not exist it wont proceed to creating extra-hosts section

Reply:
fixed by : ${this.options.ips && this.options.ips.length > 0 ?  `